docs/try/installation.md

```diff
@@ -1,159 +1,71 @@
+### Configure and compile the source Code
 
- * Open the project `RINGMesh.sln`.
- * Build the solution. 
+RINGMesh is tested under:
 
-The available compilation modes are:
+ * Linux (64bits)
+ * Windows (64 bits)
+ * Mac OS (64 bits)
 
-* Release
-* Debug
-* RelWithDebInfo 
+Please find the corresponding instruction in the [configure and compile tutorial](/try/tutorials/configure_compile_ringmesh). 
 
-Note: Visual Studio has on pile per configuration mode. You must link consistently with the expected configuration mode.
+## About pre-compiled packages
 
-### Compiling the documentation
+Pre-compiled packages are available in the [Download](/download) section.
+After having downloaded the archive that correspond to your operating system, 
+please extract the files in the folder where you want to use it.
 
-If Doxygen is installed on your computer, a target ```doc-devkit``` is built during RINGMesh configuration. You can generate the documentation by:
-* Opening the solution which is in `RINGMesh_root/build/ringmesh/RINGMesh.sln` in Visual Studio
-* Building the doc-devkit
+## Develop with RINGMesh
 
-Open the RINGMesh documentation with your favorite web browser at:  `RINGMesh_root\build\ringmesh\devkit\html\index.html`
+RINGMesh is a library that provide a development API with functions that can be used in an external project.
+We provide an example of a project that set the dependency with RINGMesh and use some functions from the RINGMesh API.
 
-## Mac OS
+Please Look at the [RINGMesh Plugin example on GitHub](https://github.com/ringmesh/RINGMeshPluginExample).
 
-### Configuring RINGMesh
+Once your project is started, you can follow some [tutorials](/try/tutorials) to learn more about the RINGMesh API.
 
-#### Using clang (without Xcode)
-As in Linux.
+## RINGMesh Utilities 
 
-#### Using Xcode IDE
-As in Windows but with the Xcode generator
-(use "-G Xcode" if you use cmake in command lines).
+To access the utilities RINGMesh need to be configured and compiled with the corresponding option.
+If you did not check that option during the configuration using CMake please go back to the 
+[configure and compile tutorial](/try/tutorials/configure_compile_ringmesh) and select the `RINGMESH_WITH_UTILITIES` option.
+Pre-compiled packages have been built with the `RINGMESH_WITH_UTILITIES` option. 
 
-### Compiling RINGMesh
-You need to install the Mac OS "Command Line Developer Tools".
+After building RINGMesh with utilities a set of executable are generated and can be execute from the RINGMesh install directory:
 
-Note: you need gcc/g++ version higher or equal to 4.2 to compile RINGMesh.
-In Mac OS, clang is used.
+ * Window: `RINGMesh\build\ringmesh\bin\CONFIG\**.exe`
+ * Linux: `RINGMesh\build\ringmesh\CONFIG\bin\**.exe`
+  
+Here is the list of the RINGMesh utilities:
 
-#### Using clang (without Xcode)
-As in Linux except for the packages.
+ * ringmesh-convert: load a GeoModel, check its validity and export GeoModel. 
```

Comment:
Sometimes you start with an uppercase, sometimes with a lowercase (for the word "load").